src/FirebaseArduino.h

```diff
@@ -24,18 +24,97 @@
 #define FIREBASE_JSONBUFFER_SIZE 200
 #endif // FIREBASE_JSONBUFFER_SIZE
 
+/**
+ * Gateway class for Arduino clients to interact with the Firebase backend.
+ * This implementation is designed to follow Arduino best practices and favor
+ * simplicity over all else. 
+ * For more complicated usecases and more control see the Firebase class in 
+ * Firebase.h.
+ */
 class FirebaseArduino {
  public:
+  /**
+   * Must be called first. This sets the class up for use.
+   * \param host Your firebase db domain name, usually X.firebaseIO.com.
```

Comment:
s/domain/host/ 

I think we also support path suffix.
als downcase the hostname.